client/src/components/App/Content/LandingPage/InvestigationTable/imageTooltipDisplay/imageTooltipDisplay.tsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
-//  import {Comment} from '@material-ui/icons';
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
```

Comment:
You can unite between the imports
`import {Tooltip, ClickAwayListener, IconButton, SvgIconTypeMap, OverridableComponent} from '@material-ui/core';`

Reply:
This is actually causes compile error, try it.